app/controllers/cuisines_controller.rb

```diff
@@ -2,4 +2,9 @@ class CuisinesController < ApplicationController
   def index
     @cuisines = Cuisine.all
   end
+
+  def show
+    @cuisine = Cuisine.find(params[:id])
+    @restaurants = @cuisine.restaurants
+  end
```

Comment:
given that this is now in React, you can remove all of the logic from this method